subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/ConsumerProvidedVariantFinder.java

```diff
@@ -90,6 +91,10 @@ private void findProducersFor(AttributeContainerInternal actual, AttributeContai
         }
     }
 
+    private AttributeContainerInternal inverseTransformAttributes(AttributeContainerInternal result, VariantTransformRegistry.Registration transform) {
```

Comment:
The name reads wrong. We are effectively completing the attributes, not inverting them. What about `computeRequestedAttributes`?